errno/errcode.go

```diff
@@ -1073,6 +1073,7 @@ const (
 	ErrHTTPServiceError                   = 8243
 	ErrPartitionColumnStatsMissing        = 8244
 	ErrColumnInChange                     = 8245
+	ErrPlacementPolicyNotEqual            = 8246
```

Comment:
We don't need to define a new error type & error name, use the existing `ErrTablesDifferentMetadata` is better.

Reply:
done.